internal/pipe/brew/brew.go

```diff
@@ -374,6 +374,10 @@ func dataFor(ctx *context.Context, cfg config.Homebrew, cl client.Client, artifa
 		}
 	}
 
+	if len(result.MacOSPackages) == 1 && result.MacOSPackages[0].Arch == "amd64" {
```

Comment:
does any consideration need to be made here for fat binaries?

Reply:
Hrm if by fat you mean all binaries in archive that arch name is `all` instead of `amd64` so I think we do not need to handle it.

Reply:
we dont, it will not trigger this if in that case..